src/app/zap-templates/common/ClustersHelper.js

```diff
@@ -473,8 +559,8 @@ function enhancedAttributes(attributes, globalAttributes, types)
   attributes.forEach(attribute => {
     enhancedItem(attribute, types);
     attribute.isGlobalAttribute     = globalAttributes.includes(attribute.code);
-    attribute.isWritableAttribute   = attribute.isWritable === 1;
-    attribute.isReportableAttribute = attribute.includedReportable === 1;
+    attribute.isWritableAttribute   = !!attribute.isWritable;
+    attribute.isReportableAttribute = !!attribute.includedReportable;
```

Comment:
Just curious. Was it causing any issues ?

Reply:
Yes, because this code was relying on https://github.com/project-chip/zap/issues/496 which the query  for "all attributes" does not have.